src/accordion.js

```diff
@@ -0,0 +1,8 @@
+export default function() {
+  var accord = $('.accordion__bar');
+  var list = $('.accordion__bar-ul');
+
+  accord.on('click', function() {
+    $(this).next(list).slideToggle();
```

Comment:
Look at using `siblings` rather than `next`

The `next` function can do more work than you want in some cases.
It's better to be clear and readable than having less code.